Automattic/vip-go-mu-plugins
# Update Gutenberg Ramp to v1.1

Gutenberg Ramp v1.1 includes various improvements including WordPress 5.0 compatibility. More info in the Ramp GitHub repository: https://github.com/Automattic/gutenberg-ramp

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

1. Check out PR.
2. Install & Activate Gutenberg
3. Edit the functions.php file of the active theme and enter something like:
```php
gutenberg_ramp_load_gutenberg( ['post_types' => ['page'], 'post_ids' => [85]] );
```
4. Make sure that Gutenberg is only active in `page` post type and the post with ID 85. You are allowed to [modify the arguments](https://github.com/Automattic/gutenberg-ramp#specifying-loading-criteria) for your test.
5. Verify that cookies are delicious. _(optional)_
